plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Entity } from '@backstage/catalog-model';
+import { errorApiRef, useApi } from '@backstage/core-api';
+import { GITHUB_ACTIONS_ANNOTATION } from '../useProjectName';
+import { useWorkflowRuns } from '../useWorkflowRuns';
+import React, { useEffect } from 'react';
+import { Table } from '@backstage/core';
+import { WorkflowRunStatus } from '../WorkflowRunStatus';
+import { Card, Link, TableContainer } from '@material-ui/core';
+import { generatePath, Link as RouterLink } from 'react-router-dom';
+
+const firstLine = (message: string): string => message.split('\n')[0];
+
+export type Props = {
+  entity: Entity;
+  branch?: string;
+  dense?: boolean;
+  limit?: number;
+};
+
+export const RecentWorkflowRunsCard = ({
+  entity,
+  branch,
+  dense = false,
```

Comment:
I feel like this should be dense to make it look more like our mockups:

![](https://backstage.io/blog/assets/20-05-20/Service_Catalog_MVP_service.png)

Reply:
Is that dense ? data-wise it's dense but the padding looks similar to below but I'm not sure. 
That said I'd be happy to add fields inline with that mock up in another PR.

![image](https://user-images.githubusercontent.com/1415599/93991559-5b731300-fd84-11ea-9542-6e080506a235.png)

Reply:
@iain-b that does not look better. Let's stick with `dense=false`.